package.json

```diff
@@ -23,5 +22,8 @@
     "dev": "encore dev",
     "watch": "encore dev --watch",
     "build": "encore production --progress"
+  },
+  "dependencies": {
```

Comment:
why in dependencies when all others are defined in `devDependencies` ?

Reply:
Sorry i missed this

Reply:
I've  missed this